app/api/posts/route.ts

```diff
@@ -4,15 +4,29 @@ import { NextRequest, NextResponse } from 'next/server';
 import { auth } from '../auth/[...nextauth]/auth';
 
 const dataSource = new MumblePostService(config.mumble.host);
+
+interface IParamsPosts {
+  params: {
+    offset: string;
+    limit: string;
+  };
+}
+
 export const GET = async (request: NextRequest): Promise<Response> => {
+  const searchParams = request.nextUrl.searchParams;
+  const limit = searchParams.get('limit');
+  const offset = searchParams.get('offset');
+
+  const limit1 = limit ? parseInt(limit, 10) : config.feed.defaultAmount;
```

Comment:
could you rename those variables? 
My suggestion:
- line 17: limitQueryParam 
- line 18: offsetQueryParam 
- line 20: limit
- line 21: offset

Reply:
in the profile pr this part gets improved. I will adapt it here too. 👍🏽